django/contrib/sessions/backends/db.py

```diff
@@ -14,14 +15,25 @@ class SessionStore(SessionBase):
     def __init__(self, session_key=None):
         super(SessionStore, self).__init__(session_key)
 
+    @classmethod
+    def get_session_class(cls):
```

Comment:
@timgraham , I believe `get_session_class` is more consistent with `get_form_class`, `get_storage_class` which are already in Django. However, if it's okay that these names differ, I'm absolutely okay with renaming.